recipes/libwebsockets/all/conanfile.py

```diff
@@ -418,7 +417,7 @@ def _configure_cmake(self):
         self._cmake.definitions["LWS_WITH_ALSA"] = False
         self._cmake.definitions["LWS_WITH_GTK"] = False
 
-        self._cmake.definitions["LWS_WITH_SYS_SMD"] = False if self.settings.os == "Windows" else True # New in 4.1.4
+        self._cmake.definitions["LWS_WITH_SYS_SMD"] = self.settings.os != "Windows"  # New in 4.1.4
```

Comment:
```suggestion
        if tools.Version(self.version) >= "4.1.0":
            self._cmake.definitions["LWS_WITH_SYS_SMD"] = self.settings.os != "Windows"
```

Option introduced in 4.1.0 actually:
https://github.com/warmcat/libwebsockets/compare/v4.0.21...v4.1.0#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aR117